playbooks/compliance_openscap_remove.yml

```diff
@@ -1,7 +1,10 @@
 ---
-# Placeholder playbook for disabling Compliance
-- hosts: localhost
+# This playbook will take care of all steps required to remove
+# Insights Compliance (OpenSCAP packages)
+- hosts: all
```

Comment:
I am not actually sure that `localhost` (the only entry in ansible inventory when running RHC) is matched by `all`

Reply:
I can swap these to localhost